source/js/components/nav/nav.scss

```diff
@@ -68,7 +66,7 @@
     #view-about .about {
       color: $white;
       font-weight: 900;
-      border-bottom: 2px solid $white;
```

Comment:
why is this changed?

Reply:
Since these elements are `<a>` themselves I thought using `text-decoration: underline` was a more common approach to get the underline shown?

Reply:
design has been approved by Kristina too